extensions/mssql/src/mssql.d.ts

```diff
@@ -228,7 +228,8 @@ declare module 'mssql' {
 		flat = 2,
 		objectType = 3,
 		schema = 4,
-		schemaObjectType = 5
+		schemaObjectType = 5,
+		none = 6
```

Comment:
I added this extra enum to differentiate between valid folder structures and empty fs (in case of comparison between dbs, dacpacs (and not project as target). @kisantia , @Benjin - if you feel this is an overkill, I can remove it and use schemaObjectType as the default (as used in https://github.com/microsoft/azuredatastudio/pull/21406/files#diff-63fe770ed4061d88b5579f1e141073f3f1c3515e63e7e1ea46c364c417c1afb9L1718 )

Reply:
Let's have the same default of schemaObjectType. This enum should match the DacExtractTarget enum in DacFx here: https://msdata.visualstudio.com/SQLToolsAndLibraries/_git/DacFx?path=/Source/DeploymentApi/DacExtractTarget.cs&_a=contents&version=GBmain

Reply:
Removed the new enum value. It still matches the DacFx side of code.